lib/gitlab/client/branches.rb

```diff
@@ -82,7 +88,6 @@ def create_branch(project, branch, ref)
     #
     # @param  [Integer, String] project The ID or name of a project.
     # @param  [String] branch The name of the branch to delete
-    # @return [Gitlab::ObjectifiedHash]
```

Comment:
Really?  Hmm... I see example responses for all the endpoints except `DELETE`...  Ohh, maybe the git diff is confusing you?  This doc is for the `delete_branch` method, not `create_branch`... (?)